usng4j-api/src/main/java/org/codice/usng4j/UsngCoordinate.java

```diff
@@ -38,6 +38,18 @@
  */
 public interface UsngCoordinate {
 
+  /** RegEx expressions for USNG/MGRS Zone parsing */
+  public final String ZONE_REGEX_STRING = "([1-9]|[1-5][0-9]|60)";
+
+  /** RegEx expressions for USNG/MGRS Latitude Bands parsing */
+  public final String LATITUDE_BAND_REGEX_STRING = "([C-JK-NP-X])\\W?([A-JK-NP-Z][A-JK-NP-V])?";
```

Comment:
❗️ This regex includes the capital letter 'I' in all three groups and should not. Change to
`"([C-HJ-NP-X])\\W?([A-HJ-NP-Z][A-HJ-NP-V])?"` should work.

Reply:
✏️ That this did not fail implies we need negative tests that check for lat bands with `I` (and the other excluded letters).

Reply:
Great catch!  Will change ASAP.  I think we need a lot more testing in general.